mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -196,96 +201,111 @@ properties:
           - !ruby/object:Api::Type::Array
             name: "tags"
             item_type: Api::Type::String
-            immutable: true
             description: |
               Network tags to add to the Compute Engine machines backing the Workstations.
           - !ruby/object:Api::Type::Boolean
             name: "disablePublicIpAddresses"
-            immutable: true
             description: |
               Whether instances have no public IP address.
           - !ruby/object:Api::Type::NestedObject
             name: "shieldedInstanceConfig"
             description: |
               A set of Compute Engine Shielded instance options.
-            immutable: true
+            diff_suppress_func: 'EmptyOrUnsetBlockDiffSuppress'
             properties:
               - !ruby/object:Api::Type::Boolean
                 name: "enableSecureBoot"
-                immutable: true
                 description: |
                   Whether the instance has Secure Boot enabled.
+                send_empty_value: true
               - !ruby/object:Api::Type::Boolean
                 name: "enableVtpm"
-                immutable: true
                 description: |
                   Whether the instance has the vTPM enabled.
+                send_empty_value: true
               - !ruby/object:Api::Type::Boolean
                 name: "enableIntegrityMonitoring"
-                immutable: true
                 description: |
                   Whether the instance has integrity monitoring enabled.
+                send_empty_value: true
           - !ruby/object:Api::Type::NestedObject
             name: "confidentialInstanceConfig"
             description: |
               A set of Compute Engine Confidential VM instance options.
-            immutable: true
+            # TODO(esu): To be removed once the main enable option becomes required.
+            diff_suppress_func: 'EmptyOrUnsetBlockDiffSuppress'
```

Comment:
similar to `shieldedInstanceConfig`

Reply:
Similar to above, replaced this with a custom flattener.